src/views/client-dashboard/ClientDashboard.vue

```diff
@@ -2,7 +2,18 @@
   <div class="es_client-dashboard-container bg_gray content">
     <div class="container margin_30_20">
       <div class="row bg-white">
-        <div class="col-md-3 border-right mt-4">
+        <div v-show="clientSize > 767 ? false : true" class="col-6 mt-4">
```

Comment:
I think you can use like this: `v-show="clientSize > 767"` which will always return true or false. No need to put the ternary

Reply:
Same for other places as well